components/dropdown/dropdown-opener-mixin.js

```diff
@@ -1,6 +1,12 @@
 import { getUniqueId } from '../../helpers/uniqueId.js';
 import { isComposedAncestor } from '../../helpers/dom.js';
 
+const intersectionObserver = new IntersectionObserver(entries => {
+	entries.forEach(entry => {
+		entry.target.__updateContentVisibility(entry.isIntersecting);
+	});
+}, { threshold: 0 });
```

Comment:
`threshold: 0` means the opener is considered intersecting if at least 1px is in view. `threshold: 1` would mean the entire opener would need to be in view.